src/sinks/clickhouse/convert.rs

```diff
@@ -0,0 +1,266 @@
+use std::{
```

Comment:
It looks like convert.rs , native_service.rs , native_sink.rs and parse.rs are all specific to the Native Protocol (native sink).

Perhaps we should organize this a bit. The `http_sink.rs` is a standalone file but we could add a `native` directory containing the native specific files.

```
config.rs
http_sink.rs
...
native/
     convert.rs
     mod.rs
     service.rs
     sink.rs
     parse.rs